core/src/test/java/com/datastrato/gravitino/storage/kv/TestEntityKeyEncoding.java

```diff
@@ -262,4 +272,50 @@ public void testNamespaceEncoding()
                   EntityType.COLUMN));
     }
   }
+
+  @Test
+  void testSpecialCharacterDecoder() throws IOException {
```

Comment:
I would suggest to use parameterized test to test more cases?

Reply:
Fixed